release.sh

```diff
@@ -30,4 +30,4 @@ git push
 git push origin $tag
 
 gem build fdk.gemspec
-gem push fdk-$version.gem
+gem push fdk-$version.gem
```

Comment:
This file should not change. Can you make it consistent with the master branch of public github

Reply:
will remove the extra line added to the end of the file